app/views/development_metrics/repositories.erb

```diff
@@ -36,13 +36,19 @@
             <div class="distribution-report-button mr-5">
               <%= link_to 'See details',
                           repository_time_to_second_review_prs_repository_index_path(@repository.name, { metric: { from: params&.dig(:metric, :from), to: params&.dig(:metric, :to) }}),
-                          class: 'btn btn-detail'
+                          class: 'btn btn-detail' unless @review_turnaround[:per_repository_distribution][0][:data].empty?
               %>
              </div>
            </div>
         </div>
         <div class="graph">
+          <% if !@review_turnaround[:per_repository][0][:data].empty? && !@review_turnaround[:per_repository_distribution][0][:data].empty? %>
```

Comment:
this is starting to get repeated 🤔 I'm wondering if it's a better way to have this.. or even if this should be in a presenter object so that we don't have logic in views